storybook/storybook-react/src/OrderedList/OrderedList.stories.tsx

```diff
@@ -14,10 +14,18 @@ const meta = {
   component: OrderedList,
   args: {
     children: orderedList,
+    inverseColor: false,
   },
   argTypes: {
     markers: { control: 'boolean' },
   },
+  decorators: [
+    (Story, context) => (
+      <div className={context.args.inverseColor ? 'amsterdam-docs-dark-background' : undefined}>
```

Comment:
Nice. Can you still add an initially visible example as well? For both ordered and unordered.

Reply:
Thanks, I forgot to mention that we use the same text for this prop on all components, so I’ve copied that.